packages/icons/icons.json

```
@@ -3883,5 +3883,12 @@
         "tags": "open, application, new-tab, external",
         "group": "interface",
         "codepoint": 62251
+    },
+    {
+        "displayName": "Float",
+        "iconName": "float",
```

I think we should be more specific and call this "floating-point". You will need to rename the SVG files to match the iconName here after you do that.

```suggestion
        "displayName": "Floating point number",
        "iconName": "floating-point",
```